addons/test_mail/models/test_mail_corner_case_models.py

```diff
@@ -164,7 +164,7 @@ class MailTestTrackDurationMixin(models.Model):
     _description = 'Fake model to test the mixin mail.tracking.duration.mixin'
     _name = "mail.test.track.duration.mixin"
     _track_duration_field = 'customer_id'
-    _inherit = ['mail.thread', 'mail.tracking.duration.mixin']
+    _inherit = ['mail.tracking.duration.mixin']
```

Comment:
I'm less keen on this, it's not intuitive that the tracking mixin would imply this is a thread. I'd keep them separate everywhere even if we make the mixin inherit thread.